config/advisory_templates.yml

```diff
@@ -247,57 +247,45 @@ boilerplates:
         Details on how to access this content are available at https://docs.redhat.com/en/documentation/openshift_container_platform/4.{MINOR}/html-single/updating_clusters/index#updating-cluster-cli
   microshift:
     rhsa:
-      synopsis: (impact) Red Hat build of MicroShift 4.{MINOR}.{PATCH} packages and security update
+      synopsis: Red Hat build of MicroShift 4.{MINOR}.{PATCH} security update
       topic: |
-        Red Hat build of MicroShift release 4.{MINOR}.{PATCH} is now available with updates to packages and images that fix several bugs. 
-
-        This release includes a security update for the Red Hat build of MicroShift 4.{MINOR}. 
-
-        Red Hat Product Security has rated this update as having a security impact of (impact). A Common Vulnerability Scoring System (CVSS) base score, which gives a detailed severity rating, is available for each vulnerability from the CVE link(s) in the References section.
+        Red Hat build of MicroShift release 4.{MINOR}.{PATCH} is now available with updates to packages and images that include a security update.
+        
+        Red Hat Product Security has rated this update as having a security impact of Moderate. A Common Vulnerability Scoring System (CVSS) base score, which gives a detailed severity rating, is available for each vulnerability from the CVE link(s) in the References section.
```

Comment:
```suggestion
        Red Hat Product Security has rated this update as having a security impact of (impact). A Common Vulnerability Scoring System (CVSS) base score, which gives a detailed severity rating, is available for each vulnerability from the CVE link(s) in the References section.
```
should still be whatever the OCP impact is; MicroShift is still sharing the codebase with OCP

Reply:
thanks - updated